assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -35,7 +35,10 @@ pageStartup.start();
 
 // ----- Redux Store ----- //
 
-const store = createStore(reducer, applyMiddleware(thunkMiddleware));
+const store = createStore(reducer, {
+  intCmp: getQueryParameter('INTCMP', 'gdnwb_copts_recurring_contributions_landing_default'),
```

Comment:
Right, sorry, I got confused here. I thought this was for the landing page but it's actually for the checkout pages right?

I added a default code for the landing page in #137, but it's the [wrong one](https://github.com/guardian/support-frontend/pull/137/files#diff-7aa4a12340a3f2b65e8d6fe204c12920R29)... too much copy-paste. *That's* the one I think should be `gdnwb_copts_contributions_landing_default`.

As far as the checkout pages are concerned, I'm not sure they should have a default INTCMP code. Would that actually tell us anything useful? I think perhaps we should leave that parameter off in this case...